@LinProp/LinProp.m

```diff
@@ -260,6 +260,54 @@
                 end
             end 
         end
+        function str = string(obj)
+            
+            % The plus/minus sign coded as unicode number so this
+            % source code file is not dependent on the encoding.
+            pm = sprintf(' \xB1 ');
+            
+            function varargout = dispParts(x)
+                x = reshape(x, [], 1); % Enforce a column vector so disp does not produce 'Column x' statements.
+                varargout = strsplit(strtrim(evalc('disp(x)')));
+            end
+            
+            str = cell(size(obj));
+            for ii = 1:numel(obj)
+                
+                val_real  = get_value(real(obj));
+                unc_real = get_stdunc(real(obj));
+                sign_real = ' ';
+                if val_real < 0
```

Comment:
Is there no `?: operator` in MATLAB like in C#, see https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/conditional-operator

Reply:
Unfortunately not.  :-(

The only other option would be:
```MATLAB
if (val_imag < 0) sign_imag = ' - '; else sign_imag = ' + '; end
```

Reply:
Ok in that case I would prefer the single line option.